content/docs/user-guide/project-structure/configuration.md

```diff
@@ -0,0 +1,379 @@
+# DVC Configuration
+
+Once initialized in a <abbr>project</abbr>, DVC populates its installation
+directory with [internal files], which include `.dvc/config`, the default
+configuration file.
+
+[internal files]: /doc/user-guide/project-structure/internal-files
+
+<admon type="warn">
+
+`.dvc/config` is meant to be tracked by Git and should not contain sensitive
+user info or secrets (passwords, SSH keys, etc). Other
+[config file locations](#config-file-locations) can be used as well.
```

Comment:
```suggestion
`.dvc/config` is meant to be tracked by Git and should not contain sensitive
user info or secrets (passwords, SSH keys, etc). Other
[config file locations](#config-file-locations) are not tracked by Git and can
be used for sensitive info.
```

Minor suggestion to be more direct in this explanation.

Reply:
Or use what's in the ref: `Use --local when in doubt`.

Reply:
Hmmm, didn't love either suggestion. I updated it in a different way (b7e9672ff), PTAL.

Reply:
What's your goal with this admon? I was trying to make sure that we include enough info so people know where they generally should store sensitive info. I feel like just saying that other locations can be used kind of leaves people hanging.

Reply:
Good point. We could move it into the next section (**Config file locations**)...

Reply:
Done too.

Reply:
Looks like you pushed to #4380 instead of this PR?

Reply:
Anyway, looks good there, thanks!